news/newsflash.txt

```diff
@@ -5,6 +5,8 @@
 # headings.  URL paths must all be absolute.
 Date: Item
 
+17-Jun-2021: New Blog post: <a href="/blog/blog/2021/06/17/OpenSSL3.0ReleaseCandidate/">OpenSSL 3.0 Release Candidate</a>
```

Comment:
I assume this PR won't be merged before that blog post is in place

Reply:
Correct

Reply:
That blog post is in place now.

Reply:
This PR was in fact merged straight after the beta release, but I forgot to close this PR. Closing it now.